### PR TITLE
Fix finding Docker image for ARM Linux builds

### PR DESCRIPTION
**Pull request by cbjeukendrup**
It's unclear why this is necessary; maybe docker changed something. And I also don't know if this is the solution; let's try

### CODE REVIEW

**Comment by cbjeukendrup** [comment]
Looks like it was not a bad first guess. https://github.com/musescore/MuseScore/actions/runs/11480787247

File: src/framework/musesampler/internal/musesamplerresolver.cpp [hunk 1]
```diff
@@ -85,11 +85,11 @@ InstrumentInfo findInstrument(MuseSamplerLibHandlerPtr libHandler, const AudioRe
 
 void MuseSamplerResolver::init()
 {
-    if (doInit(configuration()->userLibraryPath())) {
-        return;
-    }
+    // if (doInit(configuration()->userLibraryPath())) {
```
Comment (RomanPudashkin) [inline src/framework/musesampler/internal/musesamplerresolver.cpp:88]: You forgot to revert this change

**Review by RomanPudashkin — APPROVED** [review]
